snuba/query/conditions.py

```diff
@@ -321,3 +321,23 @@ def build_match(
             ),
         ]
     )
+
+
+def get_literals_for_column_condition(
+    col: str, exp_type: Any, conditions: Expression
+) -> Sequence[Literal]:
+    match = build_match(col, (ConditionFunctions.EQ,), exp_type)
```

Comment:
Intentionally ignoring IN condition? like projects in [1,2,3]

Reply:
`build_match` adds the IN condition.